src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,21 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  posts: Post[]
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo
+        title={post.title}
+        user={post.user}
+        body={post.body}
```

Comment:
Consider destructuring the 'post' object within the map callback, e.g., 'posts.map(({ id, title, user, body, comments }) => ('